static/js/components/OrderSummary.js

```diff
@@ -38,9 +38,8 @@ class OrderSummary extends React.Component {
     let text;
     let price = (finalPrice !== undefined && finalPrice !== null) ? finalPrice : coursePrice.price;
     if (price > 0) {
-      text = "Clicking below with link outside of the MIT MicroMasters app" +
-        " to an external website, where you will be able to complete the transaction by" +
-        " paying with a credit card.";
+      text = "Clicking Continue will link to a website outside of the MicroMasters app, where " +
+        "you will be able to complete the transaction, by paying with a credit card.";
```

Comment:
@roberthouse54 how sure are we of this wording? I'm just thinking that it sounds a bit strange, to my ear something like

"Clicking continue will take you to an external website, where you can complete the transaction with a credit or debit card."

sounds a lot more natural. thoughts?